plugin/pkg/scheduler/algorithm/priorities/taint_toleration.go

```diff
@@ -52,8 +52,8 @@ func getAllTolerationPreferNoSchedule(tolerations []v1.Toleration) (tolerationLi
 	return
 }
 
-func getTolerationListFromPod(pod *v1.Pod) ([]v1.Toleration, error) {
```

Comment:
Sure, please go ahead and do so. After dropping `error` from the return values, there is no reason to keep this method. It only confuses people.